lib/play.rb

```diff
@@ -59,7 +60,7 @@ def wrong_input
   end
 
   def position_taken_error
-    board.message = "\n       cleaThat position is taken,\n" + '    choose form the available ones'
+    board.message = "\n       That position is taken,\n" + '    choose form the available ones'
     continue
```

Comment:
You can't call **board** in this module.
You must move all the methods, from the `Play` module, inside which you called or used `board.` into the **Game** class directly.
The module **Play** like the one you created should only contains additional methods that must help the **Game** class.
**CHANGE THIS CONCEPTION**!